software/acpid.txt

```diff
@@ -0,0 +1,123 @@
+ACPID
+________________________________________________________________________________
+
+
+Prerequisites
+________________________________________________________________________________
+
+Make sure that ACPI-related kernel options are enabled. [0]
+
+
+Using busybox acpid
+________________________________________________________________________________
+
+Busybox provides a very limited acpid program. The only event that it works
+well for is closing the lid. For some users this is enough. To run
+commands when the lid is closed, create an executable script at
+/etc/acpi/LID/00000080. The example below will suspend your computer to RAM.
```

Comment:
This paragraph can be re-wrapped. Lines can be longer in other words.

Reply:
I'm not sure I understand what you mean, but I left a few long lines in the latest commit